client/src/components/OpeningTimes.jsx

```diff
@@ -15,43 +15,50 @@ const OpeningTimes = ({ currentStation }) => {
         <Day>
           Montag:
           <time>
-            {openingDay.monday[0].opening} - {openingDay.monday[0].closing}
+            {' '}
+            {openingDay.monday[0]?.opening} - {openingDay.monday[0]?.closing}
           </time>
         </Day>
         <Day>
           Dienstag:
           <time>
-            {openingDay.tuesday[0].opening} - {openingDay.tuesday[0].closing}
+            {' '}
```

Comment:
```suggestion
    
```

Reply:
same for the next rows 🐜

Reply:
This is not a mistake even if it seems like one. We have inserted this to always have a blank space there